PlayCover/Views/Settings/PlayCoverSettingsView.swift

```diff
@@ -13,7 +13,7 @@ struct PlayCoverSettingsView: View {
     @EnvironmentObject var storeVM: StoreVM
 
     private enum Tabs: Hashable {
-        case updates, ipasource, uninstall
+        case updates, ipasource, uninstall, install
```

Comment:
On my machine, the layout appears in the order Updates, Uninstall, IPA Sources, Install. Not sure if it's just a me thing or not.
<img width="462" alt="Screenshot 2022-10-19 at 22 36 36" src="https://user-images.githubusercontent.com/42140194/196843024-7467df02-d7ba-4c3a-9d98-c1316251243f.png">

Reply:
Updates, IPA Sources, Install, Uninstall makes more sense to me